toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -402,6 +406,9 @@ func handleBootLoader(baseConfigPath string, config *imagecustomizerapi.Config,
 
 	switch config.OS.ResetBootLoaderType {
 	case imagecustomizerapi.ResetBootLoaderTypeHard:
+		if config.Storage == nil {
```

Comment:
I thought storage is not necessary to `resetBootLoaderType` but the other way around is mandated (where resetBootLoaderType is a must when storage is specified), is that not the case anymore?